client/src/components/ComboBox/ComboBox.scss

```diff
@@ -0,0 +1,137 @@
+// Ensure consistent spacing across the whole component.
+// With the scrolling and show/hide of the field, correct spacing is critical.
+$spacing: theme('spacing.[2.5]');
+$spacing-sm: theme('spacing.5');
+
+.w-combobox {
+  width: min(400px, 80vw);
+  background: $color-white;
+  color: $color-input-text;
+  border-radius: theme('borderRadius.DEFAULT');
+  font-size: theme('fontSize.18');
+  box-shadow: theme('boxShadow.md');
+  outline: 10px solid transparent;
+}
+
+.w-combobox__field {
+  padding: $spacing;
+  padding-bottom: 0;
+
+  @include media-breakpoint-up(sm) {
+    padding: $spacing-sm;
+    padding-bottom: 0;
+  }
+}
+
+.w-combobox [role='combobox'] {
+  margin-bottom: $spacing-sm;
+
+  &[disabled] {
+    display: none;
+  }
+}
+
+.w-combobox__menu {
+  max-height: min(480px, 70vh);
+  overflow-y: scroll;
+}
+
+.w-combobox__optgroup {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-auto-flow: column;
+  gap: theme('spacing.[0.5]');
+  padding: $spacing;
+  padding-top: 0;
+
+  @include media-breakpoint-up(sm) {
```

Comment:
The spacing on top of the optgroup label seems a bit off on smaller screens, is this intended?

## Smaller screens

<img width="368" alt="image" src="https://user-images.githubusercontent.com/6379424/213491760-36f0648a-b5be-45d3-93a0-0c4e6498a963.png">

## Larger screens

<img width="368" alt="image" src="https://user-images.githubusercontent.com/6379424/213491974-20c974ea-4b45-4941-bb6c-2fc96f6fc98a.png">